llvm/lib/CodeGen/MachineDominators.cpp

```diff
@@ -136,15 +144,19 @@ void MachineDominatorTree::applySplitCriticalEdges() const {
 
   // Now, update DT with the collected dominance properties info.
   Idx = 0;
+  dbgs() << "critical Edge to split: " << CriticalEdgesToSplit.size();
```

Comment:
Stray debug printing?

Reply:
Oops, debug code to resolve test failure due to empty dominator tree...